scripts/DebugTest.ps1

```diff
@@ -0,0 +1,42 @@
+param
+(
+    [string] $TestName,
+    [string] $DirectoryName,
+    [switch] $NoBuild
+)
+
+# https://code.visualstudio.com/docs/editor/debugging
+# https://code.visualstudio.com/docs/editor/variables-reference
+
+if(!$TestName) {
+    throw "Select test name in VSCode before running!"
+}
+
+if(!$DirectoryName) {
+    throw "Failed to get directory name"
+}
+
+Write-Host "TestName: $TestName"
+Write-Host "DirectoryName: $DirectoryName"
+
+# Build
+if(!$NoBuild) {
+    & "$PSScriptRoot\Build.ps1"
+}
+
+# Locate test assembly
+$csProj = Get-ChildItem -Path $DirectoryName -Filter '*.csproj' | Select-Object -First 1
+if(!$csProj) {
+    throw "Failed to find CSProj in: $DirectoryName"
+}
+
+# [xml] $csProjXml = Get-Content -Path $csProj.FullName -Raw
```

Comment:
What's the intent behind this commented out code?

Reply:
That was showing how to get assembly name, I can remove it now. Originally I thought I needed assembly name but instead I just needed project name